.github/workflows/regression.yml

```diff
@@ -35,6 +35,7 @@ jobs:
       - name: Set up Node.js
         uses: actions/setup-node@v3
         with:
+          node-version: 16
```

Comment:
If this line is intended to be temporary, do we have an issue tracking the underlying issue that will remind us to remove this line?

Reply:
I've renamed #2636 to be more specific to this issue so we can continue tracking there.